lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -445,13 +448,15 @@ defmodule ExUnit.DocTest do
   defp adjust_indent(:text, [line | rest], line_no, adjusted_lines, indent, module) do
     case String.starts_with?(String.trim_leading(line), @iex_prompt) do
       true  ->
-        adjust_indent(:prompt, [line | rest], line_no, adjusted_lines, get_indent(line, indent), module)
+        adjust_indent(:prompt, [line | rest], line_no, adjusted_lines, get_indent(line, indent),
+                      module)
```

Comment:
100 chars seems too strict limit, it might sometimes make things worse I suppose.